packages/neon_framework/lib/src/theme/icons.dart

```diff
@@ -0,0 +1,134 @@
+// ignore_for_file: non_constant_identifier_names
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_material_design_icons/flutter_material_design_icons.dart';
+import 'package:neon_framework/src/utils/adaptive.dart';
+
+// Adding new Icons to this set:
+// Make sure to use the common doc comment style and add the correct preview.
+// If in doubt copy the previews from the icons you are redirecting to.
+
+/// A set of platform-adaptive icons.
+abstract final class AdaptiveIcons {
+  const AdaptiveIcons._();
+
+  /// Platform-adaptive icon for
+  /// <i class="material-icons md-36">search</i> &#x2014; material icon named "search"
+  /// and <i class='cupertino-icons md-36'>search</i> &#x2014; cupertino icon named "settings".
+  static IconData get search => !isCupertinoPlatform() ? Icons.search : CupertinoIcons.search;
```

Comment:
Can you invert all the conditions? I think that is easier to understand and also indicates better that the material icons are a fallback.

Reply:
I'm not 100% sure about that.
The flutter `PlatformAdaptiveIcon` has it exactly the same: https://api.flutter.dev/flutter/material/PlatformAdaptiveIcons/arrow_back.html

While developing I switched between the material, cupertino and adaptive and NeonIcons multiple times to find a fitting one and having them aligned is easier.
But I can see your point.

Reply:
I see, it's your choice if you want to change it or not :woman_shrugging:

Reply:
That's a clear nope